packages/daylight/start.go

```diff
@@ -335,10 +335,21 @@ func Start() {
 		if err != nil {
 			os.Exit(1)
 		}
-		//go func() {
-		//	na := service.NewNodeActualizer(service.DefaultBlockchainGap)
-		//	na.Run()
-		//}()
+
+		go func() {
```

Comment:
are you sure running that pice of code in separate goroutine is necessary, na.Run() itself start separate goroutine